app/javascript/src/NewService/components/ServiceSourceForm.jsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React from 'react'
+
+type Props = {
+  isServiceDiscoveryUsable: boolean,
+  serviceDiscoveryAuthenticateUrl: string,
+  handleFormsVisibility: (event: SyntheticEvent<HTMLInputElement>) => void
+}
+
+const ServiceSourceForm = (props: Props) => {
+  const {isServiceDiscoveryUsable, serviceDiscoveryAuthenticateUrl,
+    handleFormsVisibility} = props
+  const classNameDisabled = isServiceDiscoveryUsable ? '' : 'disabled'
+  return (
+    <form className="formtastic" id="new_service_source">
+      <fieldset className="inputs">
+        <ol>
+          <li className="radio">
+            <label htmlFor="source_manual">
+              <input type="radio" name="source" id="source_manual" value="manual" defaultChecked="defaultChecked" onChange={handleFormsVisibility}/>
+              <span className="new-service-source-input">Define manually</span>
+            </label>
+          </li>
+          <li className="radio">
+            <label htmlFor="source_discover"
+              className={classNameDisabled}>
```

Comment:
This name suggests the label is always disabled, I'd rename it something generic:
```suggestion
              className={labelClassName}>
```

Reply:
The only possible value is `disabled` so maybe is good to suggested is disabled

Reply:
I mean, the only possible values are disabled and _not_ disabled, right?

Reply:
only `"disabled"` , `className=""`will not render any class at all

Reply:
My point exactly